lrc/Pos/Lrc/Core.hs

```diff
@@ -26,25 +27,28 @@ import           Pos.Util.Util       (getKeys)
 --
 -- 2. Delegates who became richmen.
 findDelegationStakes
-    :: forall m . MonadIterator (StakeholderId, [StakeholderId]) m
-    => (StakeholderId -> m Bool)         -- ^ Check if user is issuer?
-    -> (StakeholderId -> m (Maybe Coin)) -- ^ Gets effective stake.
-    -> Coin
-    -> m (RichmenSet, RichmenStake) -- old richmen, new richmen
+    :: forall m . Monad m
+    => (StakeholderId -> m Bool)                   -- ^ Check if user is issuer?
+    -> (StakeholderId -> m (Maybe Coin))           -- ^ Gets effective stake.
+    -> Coin                                        -- ^ Coin threshold
+    -> Sink (StakeholderId, HashSet StakeholderId) -- ^ Old richmen, new richmen
```

Comment:
I think I already pointed out that I don't like this line. I want this haddock comment to be two lines below.
1. I am not sure if it works at all, because this comment is somewhere in the middle of this type. Even if it works, it looks a little weird.
2. This comment applies to `(RichmenSet, RichmenStake)`, but it makes the impression that it's about `(StakeholderId, HashSet StakeholderId)` part.